src/screens/About/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { FC } from 'react';
+import { StyleSheet } from 'react-native';
+
+import { blueLogo } from '@assets/images';
+import { useTranslation } from 'react-i18next';
+
+import { colors } from '@app/shared/lib';
+import MarkdownView from '@app/shared/ui/MarkdownView';
+import { Box, Text, ScrollView, Image } from '@shared/ui';
+
+const AboutScreen: FC = () => {
+  const { t } = useTranslation();
+
+  return (
+    <Box bg="$secondary">
+      <ScrollView h={'100%'} px={8}>
+        <MarkdownView
```

Comment:
We will probably need this component to be reused in the future. Could you please move it with markdownStyle to "shared/ui"?

Reply:
MarkdownView is already in shared. But styles should be passed as prop, because it's specific for every view.